src/components/mobile/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export function isTouch(): boolean {
+    if (window.PointerEvent) {
+        return navigator?.maxTouchPoints > 0;
+    }
```

Comment:
Is this correct condition?

Let's something like this:
```js
if (typeof navigation !== 'undefined' && navigator.maxTouchPoints > 0) {
    return true;
}
```

Reply:
I think it's more effective, thanks!